shell/components/nav/TopLevelMenu.vue

```diff
@@ -182,6 +182,11 @@ export default {
     $route() {
       this.shown = false;
     },
+    shown() {
+      // We need to reload the clusters so we can get the status from the cluster relationships.
+      // Specifically this is to determine if the cluster is in an error state.
+      this.$store.dispatch('management/findAll', { type: MANAGEMENT.CLUSTER, opt: { force: true } });
```

Comment:
Isn't this going to make an API call every time someone clicks to show the menu?

I'm not sure we want that - don't we get the updates via the watches on the resource?

Reply:
We don't get updates from the watches. It does make a request for each click.

Reply:
It looks like provisioning cluster receives updates but management cluster doesn't. I'll see if I can use provisioning cluster instead.